Common/src/config_structure.cpp

```diff
@@ -874,6 +874,10 @@ void CConfig::SetConfig_Options() {
   addEnumOption("MULTIZONE_SOLVER", Kind_MZSolver, Multizone_Map, MZ_BLOCK_GAUSS_SEIDEL);
   /*!\brief MATH_PROBLEM  \n DESCRIPTION: Mathematical problem \n  Options: DIRECT, ADJOINT \ingroup Config*/
   addMathProblemOption("MATH_PROBLEM", ContinuousAdjoint, false, DiscreteAdjoint, false, Restart_Flow, false);
+  /*!\brief MULTIPHYSICS_DISCRETE_ADJOINT  \n DESCRIPTION: Boolean whether multiphysics adjoint shall be used \n Options: NO, YES \ingroup Config */
+  addBoolOption("MULTIPHYSICS_DISCRETE_ADJOINT", MultiphysicsDiscreteAdjoint, false);
```

Comment:
No, one doesn't, it would have been just a developer's option to use the discrete adjoint multiphysics driver for single zone cases. I removed it completely probably exactly when you wrote that comment as now a multiphysics setting with only one specified solver will also do (see latest commits above).